db.py

```diff
@@ -0,0 +1,79 @@
+from api import tasks
+import db_config as config
+import psycopg2
+
+def connect():
+    """Initiate a postgres database connection using psypg2
+    Returns:
+    A psycopg2.extensions.connection object
+    """
+
+    return psycopg2.connect(
+        dbname = config.dbname,
+        user = config.dbuser,
+        password = config.dbpassword,
+        host = config.dbhost,
+        port = '5432'
+    )
+
+def get_users(conn):
+    """Grab a list of users from the database.
+    
+    Arguments:
+    conn: psycopg2.extensions.connection object
+    Returns:
+    A list of python dicts containing user data"""
+
+    """initiate a new list of users"""
+    users = []
+    """interact with database"""
+    cur = conn.cursor() 
+    cur.execute('SELECT * FROM users;')
+    """what we fetch is sloppy, so we do a for loop to manipualte each row"""
+    for row in cur.fetchall():
+        user_dictionary = db_row_to_dict_user(row)
```

Comment:
This line should call a new function named `db_row_to_dict_teams`. Please create this function :)